docs/pages/services/pyiron.md

```diff
@@ -9,18 +9,18 @@ Pyiron
 {: no_toc}
 
 
-[![hackmd-github-sync-badge](https://hackmd.io/ut4CsaYCSAq4K3XDOnGAqA/badge)](https://hackmd.io/ut4CsaYCSAq4K3XDOnGAqA)
-
 ## Table of Contents
 {: no_toc}
 
 - TOC
-{:toc}
+{:toc}  
+
+[pyiron](https://pyiron.org) is an integrated development environment (IDE) for computational materials science. It integrates several computational/analysis tools in a common platform and provides various functionalities for creation, management and sharing of your workflows. 
 
-# Deployment of pyiron workflow environment
-[This repository](https://github.com/materialdigital/pyiron_workflow_environment_deployment) provides the configuration files, needed for the deployment of a pre-configured jupyterhub to run pyiron workflows.
+## GitHub repository
+In our [GitHub repository](https://github.com/materialdigital/pyiron_workflow_environment_deployment), you obtain the configuration files, needed for the deployment of a pre-configured jupyterhub to run pyiron workflows.
 
-## Content of the repository
+### Content of the repository
 | File | Description |
```

Comment:
I believe the table doesn't render properly atm

Reply:
Done!